backend/lcfs/web/api/notional_transfer/services.py

```diff
@@ -0,0 +1,90 @@
+import math
+from logging import getLogger
+from typing import List
+from fastapi import Depends
+from datetime import datetime
+
+from lcfs.web.api.notional_transfer.repo import NotionalTransferRepository
+from lcfs.web.core.decorators import service_handler
+from lcfs.web.api.base import PaginationRequestSchema, PaginationResponseSchema
+from lcfs.db.models.compliance.NotionalTransfer import NotionalTransfer
+from lcfs.web.api.notional_transfer.schema import (
+    NotionalTransferCreateSchema,
+    NotionalTransferSchema,
+    NotionalTransfersSchema,
+    NotionalTransferTableOptionsSchema,
+    NotionalTransferFuelCategorySchema
+)
+
+logger = getLogger("notional_transfer_services")
+
+
+class NotionalTransferServices:
+    def __init__(self, repo: NotionalTransferRepository = Depends(NotionalTransferRepository)) -> None:
+        self.repo = repo
+
+    @service_handler
+    async def get_table_options(self) -> NotionalTransferTableOptionsSchema:
+        """
+        Gets the list of table options related to notional transfers.
+        """
+        fuel_categories = await self.repo.get_table_options()
+        return {"fuel_categories": [NotionalTransferFuelCategorySchema.model_validate(category) for category in fuel_categories]}
+
+    @service_handler
+    async def get_notional_transfers(
+        self, pagination: PaginationRequestSchema, compliance_report_id: int
+    ) -> NotionalTransfersSchema:
+        """
+        Gets the list of notional transfers for a specific compliance report.
+        """
+        notional_transfers, total_count = await self.repo.get_notional_transfers_paginated(pagination, compliance_report_id)
+
+        return NotionalTransfersSchema(
+            pagination=PaginationResponseSchema(
+                total=total_count,
+                page=pagination.page,
+                size=pagination.size,
+                total_pages=math.ceil(total_count / pagination.size),
+            ),
+            notional_transfers=[
+                NotionalTransferSchema.model_validate(notional_transfer) for notional_transfer in notional_transfers
```

Comment:
If there are no notional transfers, will this cause any errors? Can `NotionalTransfersSchema` handle an empty list of `notional_transfers`?